src/Template/Bake/Template/index.ctp

```diff
@@ -16,6 +16,7 @@
 <?php
 /**
   * @var \<%= $namespace %>\View\AppView $this
+  * @var \<%= $namespace %>\Model\Entity\<%= $entityClass %>[] $<%= $pluralVar %>
```

Comment:
Sorry for the confusion, using both is what I ment to suggest... should have explicitly mentioned that instead of going the lazy way with the easy to overlook `|` separator.